src/App.tsx

```diff
@@ -15,48 +16,88 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum Sort {
+  alph = 'Sort alphabetically',
+  length = 'Sort by length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  isReversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case Sort.alph:
+          return good1.localeCompare(good2);
+
+        case Sort.length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return isReversed ? preparedGoods.reverse() : preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
```suggestion
  const [sortField, setSortField] = useState(Sort.Default);
```
add it to the enum too